### ISSUE TITLE
message unauthenticated api in local working but in production not working

### ISSUE THREAD

**Issue by narmadhasugumar** [laravel/passport#957]
Getting error like this which is mentioned below

```
{
    "message": "Unauthenticated."
}
```

It is perfectly working in local environment but auth:api is not at all working in production mode 
i tried to fix the issue i face by:

- **Increasing the expiration time 
- by deleting my whole project and  re uploaded in production
- and by, changing the .htaccess file**

Can any one help me in this i am facing this problem for past 1 week but solving is not getting

**Comment by ankurk91** [comment]
Did you run
```
php artisan passport:install
```
on production ?

**Comment by narmadhasugumar** [comment]
Is that command needed as i ran at my local environment and as it is i pushed into my production server

**Comment by driesvints** [comment]
Keys are ignored: https://github.com/laravel/laravel/blob/master/.gitignore#L4

**Comment by rootmap** [comment]
Still not working

**Comment by shivanibenosoft** [comment]
> ```
> php artisan passport:install
> ```

Stiil does not work

**Comment by Raza9798** [comment]
@narmadhasugumar did you solve this issue?, if you solved please offer the solution. the same issue i am facing right now

**Comment by devmohamedamr** [comment]
I have a solution.

What was the problem?

For some reason, the Apache server has modules that ignore the Authorization header, say it's for security (and I don't doubt it).

How did I solve it?

I simply disabled the following modules in the Apache settings:

mod_cgid
mod_fcgid
mod_proxy_scgi
And in the httpd.conf file I added the line (Linux/CentOS 7.6):

SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
